README.md

```diff
@@ -33,7 +33,7 @@ npx eslint --init
 ```
 or rename a template from the templates dir
 ```sh
-cp node_modules/@digitalbazaar/eslint-config/templates/node-recommended.js ./eslint.config.js
+cp node_modules/@digitalbazaar/eslint-config/templates/node.js ./eslint.config.js
```

Comment:
Might make more sense to align the file names with what they are using?  There could be confusion here since there is a node config as well as a node-recommended one.